### ISSUE TITLE
Initialize Loss Function and Training Code

### ISSUE THREAD

**Issue by loparcog** [McMasterAI/StyleGAN#14]
Task Overview
--
This task will aim at making the loss function and training code. There are not many additional sections, so this should just be a simple copy and paste of the given sections. For an explanation on how this will work and run, please look at the "paint" textbook reading in the Teams chat, or at the [main notebook we will be referencing for this project](https://www.kaggle.com/amyjang/monet-cyclegan-tutorial).

From the Notebook
--
From the notebook this task will need everything from the "Train the CycleGAN" and "Visualize our Monet-esque photos" sections

From You
--
Nothing, really. We will be modifying this in the future but for now we will just be using this code as our base

Ideal Output
--
By the end of the task, *ideally* we would be able to train our model, but this will, only be possible once all other tasks are completed, so just having the code is alright
